src/setup.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const WP_ADMIN_USER = config.get( 'WP_ADMIN_USER' );
```

Comment:
I wonder how `config.get` will work in this case. Will it look for a config file inside this repo?

Reply:
meaning it would not work as-is in Jetpack project without specifying `NODE_CONFIG_DIR`?

Reply:
ah, scratch that. we already passing it in Jetpack. I wonder if we can get rid of these env vars completely since they are quite specific to Jetpack's infrastructure.

Reply:
Perhaps. I thought that the login credentials would also apply to to wp-admin logins for any WordPress site.